src/controls/extras/tree.coffee

```diff
@@ -170,6 +170,9 @@ class Tree extends Typed
           setTimeout(arguments.callee, 0)
 
     timer = false
+    bandsCountId = "bands-count-" + $('.bands-count').length
```

Comment:
@mightynimble - if you mean the collection ID, then that won't work in all cases unfortunately.  This fix was so the user could click 'Add Access Option' for a single collection - meaning you can have more than of these trees on the page simultaneously for the same collection.  Since the collection ID would be used in all instances, it wouldn't be unique :\

If you meant something else by 'parent' though then it could likely work, so long as it is unique for the entire page.  I used the current count of the element type as that will always be unique and will simply increment as the user adds more access options.